guides/common/modules/con_security-features.adoc

```diff
@@ -0,0 +1,42 @@
+[id="security-features_{context}"]
+= Security features in {Project}
+
+When deploying {Project}, consider implementing security features that enhance the overall security of your system.
+{Project} supports multiple security mechanisms, including FIPS mode, `fapolicyd`, and SELinux, to provide additional layers of protection.
+
+Federal Information Processing Standards (FIPS) Mode::
+{Project} supports running on FIPS-enabled {RHEL} hosts to comply with security standards for cryptographic modules. 
+However, {Project} itself does not currently support running in FIPS mode.
++
+FIPS mode enforces the use of validated cryptographic algorithms and modules.
+Enabling FIPS mode ensures that all cryptographic operations within the system adhere to strict security standards.
++
+For more information, see {RHELDocsBaseURL}9/html/security_hardening/switching-rhel-to-fips-mode_security-hardening[Switching RHEL to FIPS mode] in _{RHEL}{nbsp}9 Security hardening_.
+
+File Access Policy Daemon (`fapolicyd`)::
+By enabling `fapolicyd` on your {ProjectServer}, you can monitor and control file and directory access, providing an additional security layer.
++
+This feature helps prevent unauthorized code execution and enhances system integrity. You can enable or disable `fapolicyd` on both {Project} and {SmartProxyServer} at any time, depending on your security requirements.
++
+For additional information about `fapolicyd`, see {RHELDocsBaseURL}9/html/security_hardening/assembly_blocking-and-allowing-applications-using-fapolicyd_security-hardening[Blocking and allowing applications by using fapolicyd] in _Security hardening_.
++
+For instructions on enabling `fapolicyd`, see {InstallingServerDocURL}installing_satellite_server_in_a_connected_network_environment/installing_server_connected_{project-context}#using-fapolicyd-on-server_{project-context}[Using fapolicyd on Satellite Server] in _{InstallingServerDocTitle}_.
+
+Security-Enhanced Linux (SELinux)::
+{Project} supports SELinux, a mandatory access control system that restricts system access and reduces the risk of security vulnerabilities.
+SELinux enforces policies that define which processes can access specific system resources, providing protection against unauthorized access and exploitation.
++
+Running {Project} with SELinux in enforcing mode is recommended to maintain a high-security environment.
+It ensures that access control policies are strictly enforced, minimizing the risk of privilege escalation or unauthorized actions.
++
+Using SELinux is required for {Project} Server to be installed on your environment. For additional information, see {InstallingServerDocURL}preparing_your_environment_for_installation_{project-context}#system-requirements_{project-context}[System requirements] in _{InstallingServerDocTitle}_.
```

Comment:
Like SELinux and fapolicyd, the idea was to move it from the installation guide to the planning guide.

Also, technically it's not required but for Satellite it must be enabled to consider the system supported. It may be in permissive mode, though enforcing mode is recommended. For upstream I'd follow those recommendations.

Here the argument to move it isn't as strong because you can consider it a prerequisite.